### PR TITLE
DOCS-1359: Updated screenshots for CC Managed Clusters UI

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):
Calico Cloud latest and next

Issue:
https://tigera.atlassian.net/browse/DOCS-1359

Link to docs preview:
https://deploy-preview-503--tigera.netlify.app/calico-cloud/tutorials/calico-cloud-features/tour
https://deploy-preview-503--tigera.netlify.app/calico-cloud/get-started/connect/connect-cluster

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by ctauchen** [comment]
@breff How does this work for you?

**Review by bmckercher123 — APPROVED** [review]
LGTM

**Comment by ctauchen** [comment]
@HugoJLFerreira Please take a look. I've limited this to the CC feature work. I'll address the CE issues we discussed separately.

File: calico-cloud/operations/cluster-management.mdx [hunk 1]
```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
 
 ### Access your managed clusters
 
-Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager.
+Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager UI.
 
 ![cluster-menu](/img/calico-cloud/cluster-menu.png)
 
-Use this menu to move between your managed clusters. You can create as many managed clusters as your subscriptions allows.
+Use this menu to move between your clusters. You can connect clusters as your subscriptions allows.
```
Comment (HugoJLFerreira) [inline calico-cloud/operations/cluster-management.mdx]: IMO. I think the statement should read as "You can connect as many managed clusters as your subscriptions allows."
As the limitation of the subscription would be on the number

File: calico-cloud/operations/cluster-management.mdx [hunk 2]
```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
...
 
 ### Managed cluster views
 
-Each time you change to a different cluster, the entire Manager "view" is set for the cluster you've selected. For example, each managed cluster has unique indexes for Elasticsearch clusters, a separate instance of Kibana, and the dashboard reflects only the selected managed cluster.
+When you connect to a different cluster, the entire Manager UI view changes to reflect the selected cluster.
```
Comment (HugoJLFerreira) [inline calico-cloud/operations/cluster-management.mdx]: Hmm, ambiguous language. As multiple clusters can be connected...
I perhaps reword this to:
'when you selected a cluster from the cluster dropdown, the entire Manager UI view changes to reflect the selected cluster.'
or
'when a diferent cluster is selected from the cluster dropdown, the entire Manager UI view changes to reflect the selected cluster.'

File: calico-cloud_versioned_docs/version-3.15/operations/cluster-management.mdx [hunk 3]
```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
 
 ### Access your managed clusters
 
-Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager.
+Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager UI.
 
 ![cluster-menu](/img/calico-cloud/cluster-menu.png)
 
-Use this menu to move between your managed clusters. You can create as many managed clusters as your subscriptions allows.
+Use this menu to move between your clusters. You can connect clusters as your subscriptions allows.
```
Comment (HugoJLFerreira) [inline calico-cloud_versioned_docs/version-3.15/operations/cluster-management.mdx]: same as above

File: calico-cloud_versioned_docs/version-3.15/operations/cluster-management.mdx [hunk 4]
```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
...
 
 ### Managed cluster views
 
-Each time you change to a different cluster, the entire Manager "view" is set for the cluster you've selected. For example, each managed cluster has unique indexes for Elasticsearch clusters, a separate instance of Kibana, and the dashboard reflects only the selected managed cluster.
+When you connect to a different cluster, the entire Manager view changes to reflect the selected cluster.
```
Comment (HugoJLFerreira) [inline calico-cloud_versioned_docs/version-3.15/operations/cluster-management.mdx]: same as above

File: calico-cloud/operations/cluster-management.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-cloud/operations/cluster-management.mdx]: Agreed. Changed.

File: calico-cloud/operations/cluster-management.mdx (hunk 2 above)
Comment (ctauchen) [reply calico-cloud/operations/cluster-management.mdx]: Agreed. Changing to "When you select a different cluster from the cluster menu ..."
